test/sql/pragma/test_pragma_output.test

```diff
@@ -0,0 +1,18 @@
+# name: test/sql/pragma/test_pragma_output.test
+# description: Test last_profiling_output, detailed_profiling_output and all_profiling_output pragma
+# group: [pragma]
+
+statement ok
+CREATE TABLE integers(i INTEGER DEFAULT 1+3, j INTEGER)
+
+statement ok
+SELECT * FROM pragma_last_profiling_output()
+
+statement ok
+SELECT * FROM pragma_detailed_profiling_output()
```

Comment:
Could you add tests with some more complex queries? I have a feeling this might blow up when the vector size is set to a lower value (which also means it will blow up on more complex queries with the regular vector size).